### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by eimanip**
Hi,

First of all, thanks for the great script!

I made a couple of improvements in the script. The changes are backward-compatible. Here are the changes:

- **Use `path.join()` instead of string concatenation for making file names**: Using string concatenation is unsafe. Also, it made the output file to have analytics.js path file to be like ".//analytics.js". It would fix this.
- **Implement the ability to change the name of the gtag.js file name in CLI**: I wanted the ability to specify a different file name for the output file instead of `google-analytics-local.js`
- **Remove `tmp` directory**: It's already specified in the `.gitignore` file, however it was included in the repo.
- **Automatically make necessary directories while saving the file**: Using `fs.mkdir`, we can make sure that the specified folder is created and exists without prior need to do create it manually.

Let me know what you think.
Thanks

P.S. I made some other changes based on this branch to be able to specify the output directory. Currently, it's writing the `folder` path in the script file, which is something that I don't want. Because the changes are not backward-compatible, I didn't include them in this pull request. You can find these changes [here](https://github.com/eimanip/localga/tree/outputdir).

### CODE REVIEW

**Comment by scriptex** [comment]
Thanks, @eimanip ! I will take a look as soon as I can.

File: index.js [hunk 1]
```diff
@@ -2,16 +2,16 @@
  * External dependencies
  */
 const request = require('request-promise-native');
+const path = require('path');
```
Comment (scriptex) [inline index.js:5]: I don't want to be too picky, but can you move this line to the "Internal dependencies" section below?

Comment (eimanip) [reply index.js]: Done.
Thanks for noticing it. I'm also obsessive about these kinds of things :D My bad!

**Comment by scriptex** [comment]
The improvements are available on [NPM](http://npm.im/localga) and [Github](https://github.com/scriptex/localga/releases/tag/2.1.0) 🎉

**Pull request merged**